packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -887,6 +887,72 @@ export function singletonSchema<TScope extends string, TName extends string | nu
     readonly value: TName;
 }, Record<string, never>, true, Record<string, never>, undefined>;
 
+// @alpha @system
+export namespace System_TableSchema {
```

Comment:
Note for reviewers: this block is all system APIs. They are gross, but they are not user-facing. The `TableSchema` namespace below contains the user-facing APIs. These should be less disgusting and easier to review.